core/controlplane/config/encrypted_assets.go

```diff
@@ -479,6 +498,8 @@ func ReadRawAssets(dirname string, manageCertificates bool, caKeyRequiredOnContr
 			{name: "etcd-client.pem", data: &r.EtcdClientCert, defaultValue: nil, expiryCheck: true},
 			{name: "etcd-client-key.pem", data: &r.EtcdClientKey, defaultValue: nil, expiryCheck: false},
 			{name: "etcd-trusted-ca.pem", data: &r.EtcdTrustedCA, defaultValue: nil, expiryCheck: true},
+			{name: "api-aggregator-key.pem", data: &r.APIAggregatorKey, defaultValue: nil, expiryCheck: false},
```

Comment:
We seems to have several names that reflect the same concept;

- aggregation layer
- api aggregation layer
- apggregator
- api aggregator
- apiserver aggregator
- apiserver aggregation

In regard to consistency and readability, I'd prefer "apiserver aggregator", so that it is clearer that it has something to do with the kubernete's apiserver.

That being said, can we rename things like:

- `api-aggregator-key.pem` -> `apiserver-aggregator-key.pem`
- `api-aggregator.pem` -> `apiserver-aggregator.pem`
- In cluster.yaml, `api-aggregator` -> `apiserverAggregator`
- `APIAggregatorKey` -> `APIServerAggregatorKey`

Reply:
Added 062ce31 to show it. Please feel free to comment/revert/approve/etc!